flink-table/flink-table-storage/src/main/java/org/apache/flink/table/storage/filestore/combine/CombinePolicy.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.storage.filestore.combine;
+
+import org.apache.flink.table.data.RowData;
+import org.apache.flink.table.runtime.typeutils.RowDataSerializer;
+import org.apache.flink.table.storage.filestore.KeyValue;
+import org.apache.flink.table.storage.filestore.utils.RecordReader;
+
+import java.io.Serializable;
+import java.util.Comparator;
+
+/** Combine policy to combine multiple records with the same key. */
+public enum CombinePolicy implements Serializable {
+
+    /**
+     * Key is the full record. Value is a count which represents number of records of the exact same
+     * fields.
+     */
+    VALUE_COUNT,
+
+    /** Key is primary key (unique), value is the full record, only keep the last one. */
```

Comment:
Maybe `latest one` is more accurate?

https://ell.stackexchange.com/questions/64084/whats-the-difference-between-last-and-latest#:~:text=%22Last%22%20refers%20to%20the%20%22,can%20%22end%22%20multiple%20times.